### PR TITLE
Add netcore3.0 support

### PR DESCRIPTION
**Pull request by ekorobchenko**
Added support of netstandard2.1 for nuget packages
Added support of netcore3.0 for unit tests
Simplified conditions in project files

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
-    <NetStandardVersion>netstandard2.0</NetStandardVersion>
+    <NetCoreVersions>netcoreapp2.1;netcoreapp3.0</NetCoreVersions>
+    <NetStandardVersions>netstandard2.0;netstandard2.1</NetStandardVersions>
     <NetFrameworkVersion>net461</NetFrameworkVersion> <!-- Framework version that should be targeted alongside with .net standard -->
     <LegacyTargetFrameworks>net452;net45</LegacyTargetFrameworks> <!-- Legacy version, required by internal consumers of packages -->
     <TargetFrameworks>$(LegacyTargetFrameworks)</TargetFrameworks>
+    <IsNetStandard>false</IsNetStandard>
+    <IsNetStandard Condition="$(NetStandardVersions.Contains('$(TargetFramework)'))">true</IsNetStandard>
+    <IsNetCore>false</IsNetCore>
+    <IsNetCore Condition="$(NetCoreVersions.Contains('$(TargetFramework)'))">true</IsNetCore>
```
Comment (AndreyTretyak) [inline Directory.Build.props]: Thank you for your contribution. Really nice change, especially with these conditions. 
Could you please move them into a separate property group?

Comment (ekorobchenko) [reply Directory.Build.props]: Thanks, done.

**Comment by AndreyTretyak** [comment]
Our build pipeline currently use VS 2017 that does not support net core 3.0 I'll try to add support for it to unblock your PR

**Comment by AndreyTretyak** [comment]
You PR should build fine after merging of this PR #115 into master

**Pull request merged**